### DISCUSSION TITLE
The application is timed out

### DISCUSSION THREAD

**Discussion by TionKior** [community/community#84021] [GitHub Education] [answered]
### Hi everyone,

The email said that I could receive a reply before 1.2, but now that 1.3 still doesn't respond, what should I do :(
Is it because of the 1.1 holiday?
![image](https://github.com/community/community/assets/66189484/4b9c7690-41a5-4732-b38c-4c682227d2a2)

**Comment by faridnec** [comment]
Hey I think you just need to wait and contact them. Happened to me as well previously

**Reply by TionKior** [reply]
Thank you, it seems that I was too anxious

**Comment by samus-aran** [comment, marked as answer]
Hey @TionKior  👋

Our estimates are a guideline on how quickly verification emails are received by. However, during busy periods like the start of a new semester, it may take a bit longer. Usually we recommend to reach out to support after 5 days of no response.

Repeating what the email details you are able to check your application's live status [here](https://education.github.com/discount_requests/pack_application). 👍 

We're locking this discussion as we hope you have found what you were looking for. Thank you for helping us maintain a productive and tidy community for all our members.
